### ISSUE TITLE
[Toggler] Partly visible elements get wrong aria-expanded state

### ISSUE THREAD

**Issue by dj-rabel** [zurb/foundation-sites#11747]
## What should happen?

aria-expanded state should be determined, based on the existence of the given data-toggler class name if not in "animate mode".

## What happens instead?

In the first place, aria-expanded will be overridden, based on the elements theoretic visibility. Whether it uses animate or toggleClass to expand/collapse later on.

## Possible Solution

Like in the toggleClass method, we should check for hasClass to set aria-expanded in the init method, in case this method will be used later to expand/collapse the element.

## Test Case and/or Steps to Reproduce (for bugs)

https://codepen.io/anon/pen/yrmBPM

## Context

Google has sent us an EMail that we will be downranked soon, cause we have invisible important text on our page. We have it on our website, exactly like it is in the codepen case I attached.

## Your Environment

- Foundation version(s) used: 6.5.3
- Browser(s) name and version(s): Browser independent
- Device, Operating System and version: OS independent

## Checklist

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] There are no other issues similar to this one.
- [x] The issue title and template are correctly filled.
